### ISSUE TITLE
Change bright red tab color on Serene?

### ISSUE THREAD

**Issue by jathu** [tricinel/edge-theme-sublime-text-3#1]
First of all, thank you for this theme. It's the best thing I found that's close to Midnight on Xcode. I want to make a suggestion to change the tab color to something more subtle than a bright red:

![screen shot 2017-12-09 at 12 08 45 pm](https://user-images.githubusercontent.com/4391247/33797655-ee934ee4-dcd9-11e7-86a7-d54d98a41dd9.png)

I would suggest keeping all the tabs the same color as the background. However, the active tab has the red font color and x button, where as the inactive tabs have the darker color font and x button.

Also, do you know why my x and chevron icons are blurry?

**Comment by tricinel** [comment]
Hi @jathu!

Thanks for taking the time to comment on this and raise an issue. I'm glad you find the theme useful. I use it myself every day (the Serene version just like you).

I did notice that the icons are a bit blurry, but couldn't figure out why. I think it may be how Sketch is exporting them, so I will have a look at that.

Regarding the tabs, I wanted to clarify what you meant. At the moment, Serene has a couple of different states for the tabs:

1. File not active with no unsaved changes.
2. File not active with unsaved changes (dirty).
3. File active with no unsaved changes.
4. File active and dirty.

Then the hover states for all of these.

When the file is active, it has the reddish background and the font color is same color as the page background. When it is not active, it has the same background as the page and the font color is a brownish color.

Are you proposing that we get rid of the reddish background altogether? This would make the active and non-active files exactly the same and you would have no idea what file you are editing.

What am I missing? :)

Thanks again!

**Comment by jathu** [comment]
Yes, I think the red background is a bit much (too much contrast and too bright). I think we should keep the active tab the same color as the background and indicate the active tab by a bottom red border. Inactive tabs can have font with reduce opacity. Similar to how this theme does it: [ayu](https://github.com/dempfi/ayu).

![example](https://user-images.githubusercontent.com/4391247/33853687-d12bcc96-de8c-11e7-9ddd-ea266a2ca709.png)

Also for the icons you can use [feather icons](https://feathericons.com/). It's a great open source icon set.

Tell me what you think.

**Comment by tricinel** [comment]
Thanks for the input! I'm a bigger fan of the full background than the line underneath, but I think I can integrate both via a setting.

I'll see what I can do and let you know!

Cheers,
Bogdan

**Comment by tricinel** [comment]
Hey @jathu, I've toned down the color for the active tab, and at the same time I've also added the option to configure the way tabs are displayed.

If you set your ST option like this:

`edge_use_tab_backgrounds: false`

then you will get the little border at the bottom of the active tabs.

This option is available since v1.1.0 of the theme. At this moment, 1.0.4 is still the current one in Package Control, so have a little patience ;)

Also fixed the blurry icons btw. :)

Let me know what you think.

Cheers,
Bogdan
